config/routes.js

```diff
@@ -1,32 +1,59 @@
-const axios = require('axios');
+const axios = require("axios");
+const bcrypt = require("bcryptjs");
+const { authenticate, generateToken } = require("../auth/authenticate");
 
-const { authenticate } = require('../auth/authenticate');
+const db = require("../database/dbConfig");
 
 module.exports = server => {
-  server.post('/api/register', register);
-  server.post('/api/login', login);
-  server.get('/api/jokes', authenticate, getJokes);
+  server.post("/api/register", register);
+  server.post("/api/login", login);
+  server.get("/api/jokes", authenticate, getJokes);
 };
 
 function register(req, res) {
   // implement user registration
+  const user = req.body;
+  user.password = bcrypt.hashSync(user.password, 12);
+
```

Comment:
Your registration endpoint looks good Stefan. The code is formatting is clean and consistent. I know it seems like overkill but adding a custom error message in your catch could help out with debugging down the line. This [error-handling-lib](https://github.com/elunic/node-express-error-handler) is pretty clean 👍 Just a suggestion and something that will be more relevant on larger projects, where you could potentially have an entire file(s) holding your error messages.